src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -0,0 +1,827 @@
+        as that column is non-nullable.
+        """
+        # for config record:
+        # - `id` is the `id` grabbed from our config template record
+        # - `origin` is the `origin` grabbed from the config template
+        create_experience_config_query = text(
+            """
+            INSERT INTO privacyexperienceconfig (id, version, origin, component, name, disabled, is_default, allow_language_selection, dismissable, accept_button_label, banner_enabled, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title, banner_description, banner_title, reject_button_label, acknowledge_button_label)
+            VALUES (:id,  :version, :origin, :component, :name, :disabled, :is_default, :allow_language_selection, :dismissable, :accept_button_label, :banner_enabled, :description, :privacy_preferences_link_label, :privacy_policy_link_label, :privacy_policy_url, :save_button_label, :title, :banner_description, :banner_title, :reject_button_label, :acknowledge_button_label)
+            """
+        )
+
+        bind.execute(
+            create_experience_config_query,
+            {
+                **experience_config,
+                "version": 1,
+                "is_default": True,
+            },
+        )
+
+        # for history record:
+        # - `id` is a generated UUID
+        # - `experience_config_id` is the `id` grabbed from our config template record on disk
+        # - `origin` is the `origin` grabbed from the config template record on disk
+        create_experience_config_history = text(
+            """
+             INSERT INTO privacyexperienceconfighistory (id, experience_config_id, origin, name, disabled, version, component, is_default, allow_language_selection, dismissable, accept_button_label, banner_enabled, description, privacy_preferences_link_label, privacy_policy_link_label, privacy_policy_url, save_button_label, title, banner_description, banner_title, reject_button_label, acknowledge_button_label)
+             VALUES (:history_id, :id, :origin, :name, :disabled, :version, :component, :is_default, :allow_language_selection, :dismissable, :accept_button_label, :banner_enabled, :description, :privacy_preferences_link_label, :privacy_policy_link_label, :privacy_policy_url, :save_button_label, :title, :banner_description, :banner_title, :reject_button_label, :acknowledge_button_label)
+            """
+        )
+
+        bind.execute(
+            create_experience_config_history,
+            {
+                **experience_config,
+                "history_id": generate_record_id("pri"),
+                "version": 1,
+                "is_default": True,
+            },
+        )
+
+        # backfill language column in all experience config history records
+        update_experience_config_history_language = text(
+            """
+            UPDATE privacyexperienceconfighistory
+            SET language = 'en'
+        """
+        )
```

Comment:
Note to self, this will be able to be removed after my changes which make language nulllable on the historical records